app/components/ui/TwoColumnContentSection/TwoColumnContentSection.tsx

```diff
@@ -60,11 +65,23 @@ export const TwoColumnContentSection = ({
             : styles.imageContainer_right
         }
       >
-        <img
-          className={styles.image}
-          src={builder.image(image).url()}
-          alt={imageAlt}
-        />
+        {mediaType === "video" ? (
+          <div className={styles.videoWrapper}>
+            <ReactPlayer
+              url={videoUrl}
+              controls
+              muted
+              width="100%"
```

Comment:
I notice in the browser if you remove this prop the `#player` container still gets inline styles of width and height 100% applied. Seems like this might be redundant, but not positive.